messages/rules-command.md

```diff
@@ -1,27 +1,71 @@
+
+If you specify a set of code files (workspace), this command automatically displays only the rules that are actually relevant. For example, if your workspace contains only JavaScript files, the command doesn't show TypeScript rules. This command doesn't actually inspect the files, it uses just the file extension, such as ".js".
+
+If you've created a custom configuration file that overrides the default rule and engine properties, you can specify it with the --config-file flag. 
 
 # command.examples
 
-*DRAFT*: Command examples
+- List the recommended rules for all engines, which is the default rule selector; use concise table format for the output:
+
+    <%= config.bin %> <%= command.id %> 
+
+- List all rules for all engines:
+
+    <%= config.bin %> <%= command.id %>  --rule-selector all
+
+- List the rules for all engines that apply to the code in the folder "my-workspace" in the current folder:
+
+    <%= config.bin %> <%= command.id %>  --rule-selector all --workspace ./my-workspace
+
+- Similar to the previous example, but specify multiple workspaces. One workspace uses glob patterns (wildcards) to find only Apex class files (extension .cls) and the other is just a single file: 
+
+    <%= config.bin %> <%= command.id %>  --rule-selector all --workspace ./my-workspace --workspace ./my-other-workspace/**/*.cls --workspace ~/files/file.ts
+
+- List the ESLint rules that have a moderate severity (3) and the default RetireJS rules with any severity:
+
+    <%= config.bin %> <%= command.id %> --rule-selector eslint:3 --rule-selector retire-js:default
+
+- Similar to the previous example, but check the "code-analyzer.yml" file in the current folder for any engine or rule overrides and use them rather than their defaults. For example, if you changed the severity of an ESLint rule from moderate (3) to high (2) in your configuration file, then it will no longer be listed:
+
+    <%= config.bin %> <%= command.id %> --rule-selector eslint:3 --rule-selector retire-js:default --config-file ./code-analyzer.yml
+
+- List the details of the "getter-return" ESLint rule and the "no-inner-declarations" rule in any engine:
+
+    <%= config.bin %> <%= command.id %> --rule-selector eslint:getter-return --rule-selector no-inner-declarations --view detail
+
+- List the details of the ESLint rules that have the tag "problem" and high severity level (2):
+
+    <%= config.bin %> <%= command.id %> --rule-selector eslint:problem:2 --view detail
 
 # flags.workspace.summary
 
-*DRAFT*: Flag summary
+Set of files used to automatically narrow the list of rules based on the contents. You can specify one or more files, one or more folders, and use glob patterns (wildcards). 
```

Comment:
"Automatically" is a weird word here, particularly because it's not actually guaranteed to do anything for every engine. Maybe "to potentially narrow"?
Also, while we intend to support globs before going GA, we don't support them yet.
Also-also, the summary is what displays inline in the flags section. Do we want it to be this long?

Reply:
Agreed. Also the term "narrow" isn't technically true. The list of rules could also grow (for example in the case of eslint when we see eslint plugins that introduce new rules).

I'd recommend saying something like:
Set of folders and/or files that make up your workspace. When supplied, the rules command returns a more accurate list of the rules applicable to your workspace. You can specify one or more folders, one or more files, and use glob patterns.

Reply:
Also, maybe:

Set of files that you want to include in the code analysis.